configure.ac

```diff
@@ -6,7 +6,7 @@ dnl when going to/from release please set the nano (fourth number) right !
 dnl releases only do Wall, git and prerelease does Werror too
 dnl
 AC_INIT([GstShark],
-        [0.7.0.1],
+        [0.7.1.1],
```

Comment:
Just to clarify, why did you modify the `minor (third)`  instead of nano (fourth)?